Makefile

```diff
@@ -35,7 +35,6 @@ clean:
 	rm -rf test-coverage .coverage
 
 check: default linting
-	tests/generate.py
```

Comment:
How are the tests run now? By the nose invocation below?

(PS: can we stop using nose? it's dead upstream afaik)

Reply:
Yup, using nose below. I wasn't aware it was dead, anything to discover the test suite will do. I'll adress that.